tests/testthat/test-poisson.R

```diff
@@ -0,0 +1,26 @@
+context("poisson regression")
+
+test_that("solutions along regularization path are the same as glmnet", {
+  set.seed(1)
+
+  N=2000; p=10
```

Comment:
Try to keep good coding style here as well. Avoid `;`, use `<-` instead of `=`, add spacing between commas and so on.